lib/sorcery/protocols/oauth2.rb

```diff
@@ -8,13 +8,16 @@ def oauth_version
       end
 
       def authorize_url(options = {})
-        client = build_client(options)
-        client.auth_code.authorize_url(
+        query_params = {
```

Comment:
Why are we adding the query parameters to the options hash here? It doesn't seem to affect any other providers, and if it's specific to the generic openid provider then it should probably be encapsulated there.